src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -85,32 +85,48 @@ private boolean validateCreationDate(LocalDate creationDate, Errors errors, Stri
-            if (Objects.nonNull(trustCeasedDate)) {
-                final String errorMessage = ValidationMessages.NULL_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
-                setErrorMsgToLocation(errors, qualifiedFieldName, errorMessage);
-                ApiLogger.infoContext(loggingContext, errorMessage);
+        if (areBeneficialOwnersStillLinkedToTrusts(overseasEntitySubmissionDto, trustDataDto.getTrustId())) {
+
+            if (isTrustInvolvedInOverseasEntityFlagNull(trustDataDto.isTrustInvolvedInOverseasEntity(), errors, loggingContext)) {
+                if (Boolean.TRUE.equals(trustDataDto.isTrustInvolvedInOverseasEntity())) {
+                    // Ceased date of the trust must be 'null' as there are still Individual and/or Corporate BOs associated with this trust
+                    if (Objects.nonNull(trustCeasedDate)) {
+                        final String errorMessage = ValidationMessages.NULL_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+                        setErrorMsgToLocation(errors, qualifiedFieldName, errorMessage);
+                        ApiLogger.infoContext(loggingContext, errorMessage);
+                    }
+                } else {
+                    // Ceased date of the trust cannot be 'null' as this trust is no longer involved with the overseas entity
+                    validateCeasedDate(errors, loggingContext, trustCeasedDate, qualifiedFieldName, creationDate);
+                }
             }
         } else {
-            final LocalDate creationDate = trustDataDto.getCreationDate();
+            // Ceased date of the trust cannot be 'null' user is forced to supply this because there are no longer any Individual or Corporate BOs associated with this trust
+            validateCeasedDate(errors, loggingContext, trustCeasedDate, qualifiedFieldName, creationDate);
+        }
+    }
 
-            // Cease date of the trust cannot be 'null' as there are no longer any Individual or Corporate BOs associated with this trust
-            if (UtilsValidators.isNotNull(trustCeasedDate, qualifiedFieldName, errors, loggingContext)) {
-                DateValidators.isDateInPast(trustCeasedDate, qualifiedFieldName, errors, loggingContext);
-                DateValidators.isCeasedDateOnOrAfterCreationDate(trustCeasedDate, creationDate, qualifiedFieldName, errors, loggingContext);
-            }
+    private boolean isTrustInvolvedInOverseasEntityFlagNull(Boolean isTrustInvolvedInOverseasEntity, Errors errors, String loggingContext){
+        final String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.IS_TRUST_INVOLVED_IN_OE);
+        return UtilsValidators.isNotNull(isTrustInvolvedInOverseasEntity, qualifiedFieldName, errors, loggingContext);
+    }
+
+    private void validateCeasedDate(Errors errors, String loggingContext, LocalDate trustCeasedDate, String qualifiedFieldName, LocalDate creationDate) {
```

Comment:
So to avoid the method name clash, if you do revert that other change you made, this could be renamed `checkNonNullCeasedDate`.

Reply:
Moved name back and changed this.